BUILDING.md

```diff
@@ -1,11 +1,51 @@
+# Build-System CLI
+
+## Non-interactive
+```
+python build.py -h, --help
+
+usage: build.py [-h] --target {android,linux,macos,win32} --arch {x86,x64,arm}
+                [--vendor VENDOR] [--keep-native-libs] [--node-enabled]
+                [--docker] [--vagrant] [--sys-image SYS_IMAGE] [--no-shutdown]
```

Comment:
-v is just the short form for the [--vendor](https://github.com/eclipsesource/J2V8/pull/327/files#diff-a0d2463b1542394be0fc4b87475bec70R39) switch, I haven't yet seen a way to show both in the [python args parser](https://docs.python.org/2.7/library/argparse.html)